From 5b1919ec3d56328b3f363bc2a4f6d33cae479ccd Mon Sep 17 00:00:00 2001
From: VirajKanse <virajv2k@gmail.com>
Date: Thu, 13 Aug 2020 11:51:09 +0000
Subject: [PATCH] vendor: Include faceunlock config if available

---
 config/common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index f660bc1..488474a 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,6 +18,9 @@ PRODUCT_ENFORCE_RRO_EXCLUDED_OVERLAYS += vendor/revengeos/overlay
 DEVICE_PACKAGE_OVERLAYS += \
     vendor/revengeos/overlay/common
 
+#Motorola Faceunlock
+$(call inherit-product-if-exists, external/motorola/faceunlock/config.mk)
+
 # Enable SIP+VoIP on all targets
 PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml
-- 
2.28.0

